site/docs/src/json/themes/search-response.json

```diff
@@ -0,0 +1,63 @@
+{
+  "themes": [
+    {
+      "data": {
+        "addedBy": "richard"
+      },
+      "defaultMessages": "title=Login",
+      "id": "64773453-bb11-457b-a3d6-7475ec2259d0",
+      "insertInstant": 1564006815352,
+      "lastUpdateInstant": 1564084258150,
+      "localizedMessages": {
+        "fr": "title=Identifiant",
+        "es": "title=Iniciar sesión"
+      },
+      "name": "Orange Theme",
+      "stylesheet": "h1 {\r\n  color: orange;\r\n  text-align: center;\r\n}",
+      "templates": {
+        "accountEdit": "[#ftl/]",
+        "accountIndex": "[#ftl/]",
+        "accountTwoFactorDisable": "[#ftl/]",
+        "accountTwoFactorEnable": "[#ftl/]",
+        "accountTwoFactorIndex": "[#ftl/]",
+        "accountWebAuthnAdd": "[#ftl/]",
+        "accountWebAuthnDelete": "[#ftl/]",
+        "accountWebAuthnIndex": "[#ftl/]",
+        "emailComplete": "[#ftl/]",
+        "emailSent": "[#ftl/]",
+        "emailVerificationRequired": "[#ftl/]",
+        "emailVerify": "[#ftl/]",
+        "helpers": "[#ftl/]",
+        "index": "[#ftl/]",
+        "oauth2Authorize": "[#ftl/]",
+        "oauth2AuthorizedNotRegistered": "[#ftl/]",
+        "oauth2ChildRegistrationNotAllowed": "[#ftl/]",
+        "oauth2ChildRegistrationNotAllowedComplete": "[#ftl/]",
+        "oauth2CompleteRegistration": "[#ftl/]",
+        "oauth2Device": "[#ftl/]",
+        "oauth2DeviceComplete": "[#ftl/]",
+        "oauth2Error": "[#ftl/]",
+        "oauth2Logout": "[#ftl/]",
+        "oauth2Passwordless": "[#ftl/]",
+        "oauth2Register": "[#ftl/]",
+        "oauth2StartIdPLink": "[#ftl/]",
+        "oauth2TwoFactor": "[#ftl/]",
+        "oauth2TwoFactorMethods": "[#ftl/]",
+        "oauth2Wait": "[#ftl/]",
+        "oauth2WebAuthn": "[#ftl/]",
+        "oauth2WebAuthnReauth": "[#ftl/]",
+        "oauth2WebAuthnReauthEnable": "[#ftl/]",
+        "passwordChange": "[#ftl/]",
+        "passwordComplete": "[#ftl/]",
+        "passwordForgot": "[#ftl/]",
+        "passwordSent": "[#ftl/]",
+        "registrationComplete": "[#ftl/]",
+        "registrationSent": "[#ftl/]",
+        "registrationVerificationRequired": "[#ftl/]",
+        "registrationVerify": "[#ftl/]",
```

Comment:
This source JSON (and all the others, I suspect) is not up to date with all the templates. No `unauthorized` template for example.

Not sure of the best way to solve this, open to suggestions. May be part of follow on work.